buildSrc/src/main/kotlin/com/thefuntasty/donut/BitriseLint.kt

```diff
@@ -0,0 +1,19 @@
+package com.thefuntasty.donut
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.plugins.ExtraPropertiesExtension
+import org.gradle.kotlin.dsl.configure
+
+open class BitriseLint : DefaultTask() {
+
+    init {
+        group = "thefuntasty"
+
+        configure<ExtraPropertiesExtension> {
+            dependsOn("detekt")
+            project.subprojects.forEach {
+                dependsOn("${it.name}:ktlintCheck")
+            }
```

Comment:
+ android lint?

Reply:
I have created separate issues for this https://github.com/thefuntasty/donut/issues/15